jina/__init__.py

```diff
@@ -51,6 +51,8 @@
     'JINA_DEFAULT_HOST',
     'JINA_DISABLE_UVLOOP',
     'JINA_FULL_CLI',
+    'JINA_HUB_ROOT',
+    'JINA_HUB_CACHE_DIR',
```

Comment:
what's the difference?

Reply:
> what's the difference?

`JINA_HUB_ROOT` (default is `~/.jina/hub-packages`) is the place for installing source-code package of Jina hub executor  . `JINA_HUB_CACHE_DIR` (default is `~/.cache/jina`) is a cache folder that will be used as target  path of downloading.